packages/checkout-storefront/src/sections/Addresses/useAddressAvailability.ts

```diff
@@ -0,0 +1,27 @@
+import { CountryCode, useChannelQuery } from "@/checkout-storefront/graphql";
+
+interface UseAddressAvailabilityProps {
+  pause: boolean;
+}
+
+export const useAddressAvailability = (
+  { pause }: UseAddressAvailabilityProps = { pause: false }
+) => {
+  const [{ data }] = useChannelQuery({
+    variables: { slug: "default-channel" },
+    pause,
+  });
+
+  const availableShippingCountries: CountryCode[] =
+    (data?.channel?.countries?.map(({ code }) => code) as CountryCode[]) || [];
```

Comment:
I wonder where u seen it since we only do it once, here 😅

Reply:
Mapping country to code is also in `useCountrySelect`

Reply:
Just the code then - sure but it's two places, the countries are different data cause here it's channel countries which need to be retyped since they come in as strings from the api and there it's our hardcoded counties. The data is not changed in any way, it's literally { code } => code, so I think for those two places it'd be a bit of an overkill to write a util function that does nothing but return this one prop